cg/meta/clean/flow_cell_run_directories.py

```diff
@@ -104,15 +103,42 @@ def archive_sample_sheet(self) -> None:
             return
         LOG.info("Sample sheet found!")
         hk_bundle: Bundle = self.hk.bundle(name=self.id)
-        hk_tags: List[str] = [SequencingFileTag.ARCHIVED_SAMPLE_SHEET, self.id]
+        hk_tags: List[str] = [
+            self.id,
+        ]
```

Comment:
This can be set up into one line. Probably Black did that formatting before you modified the line